sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md

```diff
@@ -9,7 +9,7 @@ generate-model-factory: false
 csharp: true
 library-name: Hci
 namespace: Azure.ResourceManager.Hci
-require: https://github.com/Azure/azure-rest-api-specs/blob/7d1f6b268def6736833a08311c87cc96740eaf03/specification/azurestackhci/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/51b8f2084b05e705baf30f24ea0a40a346e2c5a5/specification/azurestackhci/resource-manager/readme.md
```

Comment:
As I can see, the latest readme should be https://github.com/Azure/azure-rest-api-specs/blob/3eaf771dc5323403a3144a8f731083b7d9ea4f97/specification/azurestackhci/resource-manager/readme.md

Merge the latest main branch to your branch, and try to regenerate code.

Reply:
Done